### PR TITLE
Add separator in wizard stage 4

### PR DESCRIPTION
**Pull request by wantedsystem**
Result =>

Capture d’écran 2021-11-24 à 20 29 53

### CODE REVIEW

File: src/newLaunch/baseStage.scss [hunk 1]
```diff
@@ -1,6 +1,11 @@
 @import "styles/styles.scss";
 // This styles are common to all the stages components
 // Theme for the stages
+
+.dividerFundings {
+  background-color: #3e3351;
```
Comment (dkent600) [inline src/newLaunch/baseStage.scss:6]: Please always use the color variables from _colors.scss

File: src/newLaunch/lbp/stages/stage4.html [hunk 2]
```diff
@@ -76,7 +76,7 @@
         <div class="infoInput maxTarget fundingTarget">
 
           <div class="fundingDescrp">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit
```
Comment (dkent600) [inline src/newLaunch/lbp/stages/stage4.html:79]: I don't think we were supposed to remove this text.  Please check with Renc about this.

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newLaunch/lbp/stages/stage4.html (hunk 2 above)
Comment (wantedsystem) [reply src/newLaunch/lbp/stages/stage4.html:79]: @dkent600 i asked @renc-curvelabs  and said that we need to remove it like i did

Comment (dkent600) [reply src/newLaunch/lbp/stages/stage4.html:79]: ok

**Review by dkent600 — APPROVED** [review]

**Pull request merged**